components/SplitsDisplayTab.vue

```diff
@@ -9,7 +9,7 @@
         Graphs' Y axis starts at zero
       </loading-switch>
       <loading-switch v-model="graphCurrentAttemptHline" class="mb-2">
-        Display current attempt times' as an horizontal line
+        Display current attempt times as a horizontal line
```

Comment:
It seems I was drunk or very tired when I wrote that but what I actually meant was this
```suggestion
        Display current attempt's times as a horizontal line
```